asset-transfer-basic/rest-api-typescript/.env.sample

```diff
@@ -0,0 +1,39 @@
+LOG_LEVEL=info
```

Comment:
Worth adding comments to this file?

Reply:
Is this ok? https://github.com/hyperledgendary/fabric-rest-sample/blob/main/asset-transfer-basic/rest-api-typescript/.env.sample#L1-L7